api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -202,6 +203,9 @@ message HttpConnectionManager {
 
     // Whether unix socket addresses should be considered internal.
     bool unix_sockets = 1;
+
+    // list of cidr_ranges that are treated as internal.
+    repeated config.core.v3.CidrRange cidr_ranges = 2;
```

Comment:
Please add a version history entry as well.

Reply:
updated `current.rst`, PTAL when you have a moment @htuch. Thanks!